ApplyToBecomeInternal/ApplyToBecomeInternal/Models/ApplicationForm/Sections/AboutConversionSection.cs

```diff
@@ -21,37 +21,57 @@ public AboutConversionSection(ApplyingSchool application) : base("About the conv
 			new[]
 			{
 				new FormField("The name of the school", application.SchoolName)
-			};
-		private IEnumerable<FormField> GenerateContactFields(ApplyingSchool application) =>
-			new[]
+			};	
+		private IEnumerable<FormField> GenerateContactFields(ApplyingSchool application)
+		{
+			var formFields = new List<FormField>();
+			formFields.Add(new FormField("Name of headteacher", application.SchoolConversionContactHeadName));
+			formFields.Add(new FormField("Headteacher's email address", application.SchoolConversionContactHeadEmail));
+			formFields.Add(new FormField("Headteacher's telephone number", application.SchoolConversionContactHeadTel));
+			formFields.Add(new FormField("Name of the chair of the Governing Body", application.SchoolConversionContactChairName));
+			formFields.Add(new FormField("Chair's email address", application.SchoolConversionContactChairEmail));
+			formFields.Add(new FormField("Chair's phone number", application.SchoolConversionContactChairTel));
+			formFields.Add(new FormField("Approver's name", application.SchoolConversionApproverContactName));			
+			formFields.Add(new FormField("Approver's email address", application.SchoolConversionApproverContactEmail));
+			if (application.SchoolConversionContactRole.ToUpper()== "OTHER")
 			{
-				new FormField("Name of headteacher", application.SchoolConversionContactHeadName),
-				new FormField("Headteacher's email address", application.SchoolConversionContactHeadEmail),
-				new FormField("Headteacher's telephone number", application.SchoolConversionContactHeadTel),
-				new FormField("Name of the chair of the Governing Body", application.SchoolConversionContactChairName),
-				new FormField("Chair's email address", application.SchoolConversionContactChairEmail),
-				new FormField("Chair's phone number", application.SchoolConversionContactChairTel),
-				new FormField("Approver's name", application.SchoolConversionApproverContactName),
-				new FormField("Approver's email address", application.SchoolConversionApproverContactEmail)          
-			};
+				formFields.Add(new FormField("Telephone number", application.SchoolConversionMainContactOtherTelephone));
+				formFields.Add(new FormField("Their role", application.SchoolConversionMainContactOtherRole));
+			}			
```

Comment:
still missing?
applyingSchools.schoolConversionMainContactOtherName
applyingSchools.schoolConversionMainContactOtherEmail

Reply:
if we have "Other" option we got 2 extract fields which has been covered in the if condition, if i added again approvename and email in if condition there will be 2 approversname and email addresses